int32 transceiverID
int32 responseID
string transceiverModelName
string data